### PR TITLE
Validate that the new standardised Cypress testing article works for us

### LINKED ISSUE #3146: Validate that the new standardised Cypress testing article works for us
**Is your feature request related to a problem? Please describe.**
In order to make cross-service testing easier, we are planning to create a standardised test article.

**Describe the solution you'd like**
An asset has been created on the test environment.  This task is to evaluate whether it contains all of the features necessary for our tests.

The asset is cn7k01xp8kxo

Dev to pair with tester (James B) to complete issue.

**Testing notes**

- [x] - We need to check that the article doesn't 404
- [x] - It has all the agreed components
- [x] - Existing Cypress tests pass
- [x] - Any components (excluding video and audio) that do not have Cypress tests will require new e2es
- [ ] 

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: Rachael Njeri <rachael.njeri@andela.com>

Resolves #3146

**Overall change:** _Update the Cypress config to use the new asset._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by RayNjeri** [comment]
I have validated the following:

The following components are pressent 
- [x] Headline, 
- [x] Image with caption and BBC copyright, 
- [x] Text block/Paragraph, 
- [x] Inline link (internal), 

All tests pass. cc: @jamesbrumpton 

Coverage for `image with caption` is missing though

**Comment by jimjohnsonrollings** [comment]
Asset on live: cj7xrxz0e8zo

**Review by EinsteinNjoroge — APPROVED** [review]
LGTM

**Review by HarveyPeachey — APPROVED** [review]
Ran on local and LGTM

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -786,9 +786,10 @@ module.exports = {
     pageTypes: {
       articles: {
         path:
+          // eslint-disable-next-line no-nested-ternary
```
Comment (rhenshaw56) [inline cypress/support/config/services.js]: what was the reason to disable linting here ??
I don't think you need it

Comment (RayNjeri) [reply cypress/support/config/services.js]: Oh, shoot, had an earlier implementation that was nested, missed to remove the comment after i reverted. Thanks for taking note of this

**Review by rhenshaw56 — APPROVED** [review]
Looks good.

**Comment by jamesdonoh** [comment]
@RayNjeri have tried running the full tests (not smoke) against test and live to verify it works across all environments?

**Review by jamesdonoh — APPROVED** [review]
Thanks for this 👍

**Pull request merged**